### ISSUE TITLE
Delete could be a bit more friendly

### ISSUE THREAD

**Issue by dlannan-kakutai** [heroiclabs/nakama#638]
## Description
When editing storage data in the nakama console, sometimes when deleting objects it will automatically delete all linked objects without notification. I think adding a little "You have X number of linked items, Are you sure?" or something similar would really help with storage data editing. 

## Steps to Reproduce
Example:
1. Create some storage data (import or api)
2. Create connected storage items (via users)
3. Delete a user with multiple storage items associated with it.
4. All data is deleted that is associated with the user without notification.

## Expected Result
A popup giving some notice about the large changes about to occur.
I think for small changes, this it is fine, its more important if an admin _accidentally_ deletes an unknown user that may be critical in the storage system

## Context
- [ x ] Defold

## Your Environment
- Nakama: 3.2.1+6790d453
- Database: Cocroach
- Docker - used the docker-compose instructions from the document
- Ubuntu 18.04

**Comment by novabyte** [comment]
Thanks for the feedback @dlannan-kakutai. We don't delete storage objects that are "linked" to each other (we have no concept in the storage engine API). We do delete all objects owned by a user if the user is deleted. Is this the scenario you encountered?

**Comment by dlannan-fmad** [comment]
Yes. I believe so. The storage page was selected, then the storage item - example below
![image](https://user-images.githubusercontent.com/60128766/124549196-f8172500-de6d-11eb-984c-fd86c4697056.png)
Then the key is selected in the url which goes to this page:
![image](https://user-images.githubusercontent.com/60128766/124549244-0d8c4f00-de6e-11eb-831c-b35dfbe70c75.png)
And delete was selected. Because its a single line item, it was expected the store would be removed it removed other stores too (I think all of them actually). I'll try to make an empty example of what was done so its repeatable (when I get some free time - hopefully).

I think maybe some info at least about what is going to happen would be useful. Or.. better still maybe configurable? So that users have the option of having some notifications of what is happening. 

Im less concerned about what happens, and more about handing this over to other admin people and ending up with a bit of a maintenance nightmare.

**Comment by jpntex** [comment]
New confirmation dialogs released with the new Console UI on [v3.30.0](https://github.com/heroiclabs/nakama/releases/tag/v3.30.0)
